nautobot_chatops/api/views/webex.py

```diff
@@ -16,20 +16,34 @@
 from webexteamssdk.exceptions import AccessTokenError, ApiError
 
 from nautobot_chatops.workers import get_commands_registry, commands_help, parse_command_string
-from nautobot_chatops.dispatchers.webex_teams import WebExTeamsDispatcher
+from nautobot_chatops.dispatchers.webex import WebExDispatcher
 from nautobot_chatops.utils import check_and_enqueue_command
 
 logger = logging.getLogger(__name__)
 
-TOKEN = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_teams_token"]
+# v1.4.0 Deprecation warning
+if (
+    "webex_teams_token" in settings.PLUGINS_CONFIG["nautobot_chatops"]
+    and "webex_token" not in settings.PLUGINS_CONFIG["nautobot_chatops"]
+):
+    TOKEN = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_teams_token"]
+    logger.warning("The 'webex_teams_token' setting is deprecated, please use 'webex_token' instead")
+else:
+    try:
+        TOKEN = settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_token"]
+    except KeyError as err:
+        ERROR_MSG = "The 'webex_token' setting must be configured"
+        logger.error(ERROR_MSG)
+        raise KeyError(ERROR_MSG) from err
```

Comment:
I am concerned about this, as it seems like this would crash the chatops plugin. This is different than the previous that would only warn if the token was missing. This wouldn't matter if webex was disabled as this file is still loaded.

Reply:
In testing, this didn't seem to cause the error like I thought, but it is different than the previous code that just emitted a warning.  I think this is due to the default_setting being `None`.  Which makes me wonder if this code is even going to be executed.

Reply:
I believe even if this threw an error, we would want it to crash.  The Try/Except statement is to log it properly before crashing.
It's saying if the Webex client is being loaded, but no token is set for either the old name or new name, log the error and fail out.

Reply:
I am not a huge fan of letting an app crash as far as user experience goes.  But, this code will never be executed due to the plugin defaults.